icechunk/src/virtual_chunks.rs

```diff
@@ -122,8 +129,25 @@ pub fn mk_default_containers() -> HashMap<ContainerName, VirtualChunkContainer>
                 store: ObjectStoreConfig::LocalFileSystem(PathBuf::new()),
             },
         ),
+        (
+            "http".to_string(),
+            VirtualChunkContainer {
+                name: "http".to_string(),
+                url_prefix: "http".to_string(),
+                store: ObjectStoreConfig::Http(HashMap::new()),
+            },
+        ),
+        (
+            "https".to_string(),
```

Comment:
I guess we don't need to add `https` because `http` will match..

Reply:
done